tests/lwip_sock_udp/main.c

```diff
@@ -1288,6 +1288,38 @@ static void test_sock_udp_send6__socketed(void)
     expect(_check_net());
 }
 
+static void test_sock_udp_sendv6__socketed(void)
+{
+    static const ipv6_addr_t src_addr = { .u8 = _TEST_ADDR6_LOCAL };
+    static const ipv6_addr_t dst_addr = { .u8 = _TEST_ADDR6_REMOTE };
+    static const sock_udp_ep_t local = { .addr = { .ipv6 = _TEST_ADDR6_LOCAL },
+                                         .family = AF_INET6,
+                                         .netif = _TEST_NETIF,
+                                         .port = _TEST_PORT_LOCAL };
+    static const sock_udp_ep_t remote = { .addr = { .ipv6 = _TEST_ADDR6_REMOTE },
+                                          .family = AF_INET6,
+                                          .port = _TEST_PORT_REMOTE };
+
+    const iolist_t tail = {
+        .iol_base = "EFGH",
```

Comment:
this would now be needlessly placed in RAM if `tail` were not `const` itself

Reply:
quick testing shows they are with our cortex-m gcc